packages/ocamlapi/ocamlapi.0.0.1/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+maintainer: "stephanos.tsoucas@gmail.com"
+authors: ["Stephanos Tsoucas"]
+license: "Apache-2.0"
+synopsis: "Path-based HTTP request routing for Ocaml"
+homepage: "https://github.com/nosman/Ocamlapi"
+bug-reports: "https://github.com/nosman/Ocamlapi/issues"
+dev-repo: "git+https://github.com/nosman/Ocamlapi.git"
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.04"}
+  "ounit"
+  "dune"
```

Comment:
```suggestion
  "dune" {build}
```
This is most probably a build dependency and should be marked as is to avoid unnecessary recompilations

Reply:
(same thing with the other packages)